<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VcrossAddPlotDialog</class>
 <widget class="QDialog" name="VcrossAddPlotDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vertical Cross Sections -- Add Plot</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stack">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="modelPage">
      <layout class="QVBoxLayout" name="layoutModels" stretch="0,0,1">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="modelLabel">
         <property name="text">
          <string>Choose a &amp;model:</string>
         </property>
         <property name="buddy">
          <cstring>modelFilter</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="modelFilter">
         <property name="placeholderText">
          <string>type to filter models</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="modelList">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="reftimePage">
      <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,1">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="reftimeLabelModel">
         <property name="text">
          <string notr="true">Chosen model: %1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="reftimeLabel">
         <property name="text">
          <string>Choose a r&amp;eference time:</string>
         </property>
         <property name="buddy">
          <cstring>reftimeList</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="reftimeList">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="plotPage">
      <layout class="QVBoxLayout" name="layoutFields" stretch="0,0,0,0,1">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="plotLabelModel">
         <property name="text">
          <string notr="true">Chosen model: %1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="plotLabelReftime">
         <property name="text">
          <string notr="true">Chosen reference time: %1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="plotLabel">
         <property name="text">
          <string>Choose &amp;plot(s):</string>
         </property>
         <property name="buddy">
          <cstring>plotFilter</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="plotFilter">
         <property name="placeholderText">
          <string>type to filter plots</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="plotList">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::MultiSelection</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRestart">
       <property name="text">
        <string>&amp;Restart</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonBack">
       <property name="text">
        <string>&amp;Back</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonNext">
       <property name="text">
        <string>&amp;Next</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonAdd">
       <property name="text">
        <string>&amp;Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
